darwin/dataset/local_dataset.py

```diff
@@ -10,6 +10,49 @@
 
 
 class LocalDataset(object):
+    """
+    Creates a dataset locally.
```

Comment:
```suggestion
    Base class representing a V7 Darwin dataset that has been pulled locally already.
    It can be used with PyTorch dataloaders. See ``darwin.torch`` module for more specialized dataset classes, extending this one.
```